tasks/rh-sign-cosign-image/tests/test-rh-sign-image-multiple-components.yaml

```diff
@@ -0,0 +1,134 @@
+              {
+                "sign": {
+                  "request": "hacbs-signing-pipeline",
+                  "configMapName": "signing-config-map"
+                }
+              }
+              EOF
+    - name: run-task
+      taskRef:
+        name: rh-sign-cosign-image
+      params:
+        - name: snapshotPath
+          value: snapshot_spec.json
+        - name: dataPath
+          value: data.json
+        - name: secretName
+          value: 'test-rh-sign-cosign-image-secret'
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      runAfter:
+        - setup
+    - name: check-result
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        steps:
+          - name: check-result
+            image: quay.io/redhat-appstudio/release-service-utils:447ea0580a2cdd48b4091e1df86fab5c3f86d01c
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+              echo "check results"
+              _TEST_PUB_REPO="registry.redhat.io/test-product/test-image0"
+              _TEST_REPO="quay.io/test-product----test-image0"
+
+              EXPECTED=$(cat <<EOF
+              inspect --raw docker://${_TEST_REPO}@sha256:1111
+              inspect --raw docker://${_TEST_REPO}@sha256:0000
+              EOF
+              )
+              CALLS=$(cat "$(workspaces.data.path)/mock_skopeo_calls")
+              [ "$CALLS" = "$EXPECTED" ] || exit 1
+              CALLS=$(cat "$(workspaces.data.path)/mock_cosign_calls")
+              PRE_ARGS="AWS_REGION=us-test-1 AWS_ACCESS_KEY_ID=test-access-key AWS_SECRET_ACCESS_KEY=test-secret-access-key"
+              COMMON_ARGS="-t 3m0s sign --key aws://arn:mykey"
+              EXPECTED=$(cat <<EOF
+              $PRE_ARGS cosign $COMMON_ARGS --sign-container-identity ${_TEST_PUB_REPO}@sha256:1111-1 ${_TEST_REPO}@sha256:1111-1
+              $PRE_ARGS cosign $COMMON_ARGS --sign-container-identity ${_TEST_PUB_REPO}@sha256:1111-2 ${_TEST_REPO}@sha256:1111-2
+              $PRE_ARGS cosign $COMMON_ARGS --sign-container-identity ${_TEST_PUB_REPO}@sha256:1111-3 ${_TEST_REPO}@sha256:1111-3
+              $PRE_ARGS cosign $COMMON_ARGS --sign-container-identity ${_TEST_PUB_REPO}@sha256:1111 ${_TEST_REPO}@sha256:1111
+              $PRE_ARGS cosign $COMMON_ARGS --sign-container-identity ${_TEST_PUB_REPO}@sha256:0000-1 ${_TEST_REPO}@sha256:0000-1
+              $PRE_ARGS cosign $COMMON_ARGS --sign-container-identity ${_TEST_PUB_REPO}@sha256:0000-2 ${_TEST_REPO}@sha256:0000-2
+              $PRE_ARGS cosign $COMMON_ARGS --sign-container-identity ${_TEST_PUB_REPO}@sha256:0000-3 ${_TEST_REPO}@sha256:0000-3
+              $PRE_ARGS cosign $COMMON_ARGS --sign-container-identity ${_TEST_PUB_REPO}@sha256:0000 ${_TEST_REPO}@sha256:0000
+              EOF
+              )
+              [ "$CALLS" = "$EXPECTED" ] || exit 1
+      runAfter:
+        - run-task
+  finally:
```

Comment:
I don't think we need this task in the test. The workspace is just a volume claim that will be released after the pipeline finishes IIUC. At least, we don't clear out the workspace at the end of our other test files (we do remove the CRs, but no CRs are created here)